packages/common/tests/aws.js

```diff
@@ -97,3 +99,50 @@ test('downloadS3File resolves filepath if key is found', async (t) => {
 
   t.is(result, Body);
 });
+
+test('pullStepFunctionEvent returns original message if message not on S3', async (t) => {
+  const event = {
+    cumulus_meta: {
+      state_machine: 'state machine',
+      execution_name: 'execution'
+    },
+    meta: {
+      bucket: 'test bucket',
+    }
+  };
+
+  const message = await aws.pullStepFunctionEvent(event);
+
+  t.deepEqual(message, event);
+});
+
+test('pullStepFunctionEvent returns message from S3', async (t) => {
+  const fullMessage = {
+    cumulus_meta: {
+      state_machine: 'state machine',
+      execution_name: 'execution'
+    },
+    meta: {
+      bucket: 'test bucket'
+    }
+  };
+
+  const event = {
+    cumulus_meta: {
+      state_machine: 'state machine',
+      execution_name: 'execution'
+    },
+    replace: {
+      Bucket: 'test bucket',
+      Key: 'key'
+    }
+  };
+
+  const stub = sinon.stub(aws, 'getS3Object').resolves({ Body: JSON.stringify(fullMessage) });
+
+  const message = await aws.pullStepFunctionEvent(event);
+
+  t.deepEqual(message, fullMessage);
+
+  stub.restore();
```

Comment:
Would it make sense to move this `stub.restore()` into a try-finally block? I don't think things would break the way they are now, especially since a failed test would cause the suite to fail, but were we to move to something like what we have for Jasmine where the suite continues after failures, it would be good to have tests clean up after themselves.

Reply:
Done